modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -28,45 +33,94 @@ func (suite *KeeperTestSuite) TestTrySendTx() {
 		expPass  bool
 	}{
 		{
-			"success", func() {
-			}, true,
+			"success",
+			func() {
+				interchainAccountAddr, found := suite.chainB.GetSimApp().ICAKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID)
+				suite.Require().True(found)
+
+				msg := &banktypes.MsgSend{
+					FromAddress: interchainAccountAddr,
+					ToAddress:   suite.chainB.SenderAccount.GetAddress().String(),
+					Amount:      sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))),
```

Comment:
```suggestion
					Amount:      sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(100))),
```

Reply:
Can do the same elsewhere in the test as well :)

Reply:
100%, I only discovered this default var when reviewing @charleenfei's PR. Was going to update to align :)

Reply:
Done! 

I've also updated hardcoded strings `"channel-0"` and `"connection-0"` that are used throughout tests, with `ibctesting.FirstChannelID` and `ibctesting.FirstConnectionID` on my submodules branch.